src/environment.rs

```diff
@@ -0,0 +1,103 @@
+/// Structure to deal with environment variables
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct Environment {
+    /// Customized environment variables
+    pub vars: Vec<(String, String)>,
+    /// Define if the structure must inherit
+    pub inherit: bool,
+}
+
+impl Default for Environment {
+    fn default() -> Self {
+        Self {
+            vars: vec![],
+            inherit: false,
+        }
+    }
+}
+
+impl Environment {
+    /// Create a new Environment object with inherit
```

Comment:
I feel like a little more context would be helpful
`Create a new Environment that inherits this process' environment.`